src/utils.ts

```diff
@@ -24,23 +24,23 @@ export function readKeypair() {
   );
 }
 
-// Sleep Time with some slight randomness to the time
+// Sleep time with some slight randomness to the time
 export function sleepRandom(period: number) {
-  const randomDrift = (Math.random() * 2 - 1) * period * percentDrift;
+  const randomDrift = (Math.random() * 2 - 1) * period;
```

Comment:
Without the percent drift we default to using 100% so L31 results in larger timeout than intended